### DISCUSSION TITLE
Is there a pdm equivalent to pip's `--no-cache-dir` flag?

### DISCUSSION THREAD

**Discussion by ryansheabla** [pdm-project/pdm#2476] [Q&A]
Running into some issues with mysqlclient, which can be easily fixed by using:

`pip install --no-cache-dir mysqlclient`

Is there a way to do this in pdm?

**Comment by anentropic** [comment]
exact same problem here... Homebrew upgraded mysql for me while installing something else and now mysqlclient is broken

have deleted my virtualenv and reinstalled but I the same errors afterwards, feels like I probably need to force pip to recompile or whatever, not clear how to do this in a pdm project though

**Reply by anentropic** [reply]
if your venv is in the default pdm location you can fix this by:

```
.venv/bin/python -m ensurepip
.venv/bin/python -m pip uninstall mysqlclient
.venv/bin/python -m pip install --no-cache-dir mysqlclient
```

**Reply by anentropic** [reply]
However, if you subsequently `rm -rf .venv` and `pdm install` then the problem will come back ☹️ 

alternative fix that is better(worked for me):
```
rm -rf .venv
pdm cache clear
pdm install
```
